### PR TITLE
Align files

### PR DESCRIPTION
**Pull request by architectbot**
This PR is generated by https://github.com/giantswarm/github, it will be merged automatically once all requirements are met

### CODE REVIEW

**Comment by ubergesundheit** [comment]
@mleisa check changes I did in https://github.com/giantswarm/velero-app/pull/43
please do the same here (add a new `_helpers.tpl` and supply common and selector labels from there)

File: helm/keda/templates/12-keda-deployment.yaml [hunk 1]
```diff
@@ -5,13 +5,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     app: {{ .Values.operator.name }}
-    name: {{ .Values.operator.name }}
```
Comment (mleisa) [inline helm/keda/templates/12-keda-deployment.yaml:8]: Was this "name" an additional label or metadata that was indented wrong?

Comment (ubergesundheit) [reply helm/keda/templates/12-keda-deployment.yaml:8]: I think this was a label, the `name` is already there in line 4 above.

**Review by ubergesundheit — CHANGES_REQUESTED** [review]
I don't see the `"labels.selector"` used except in the `"labels.common"` helper.

The idea behind defining a selector labels helper is to have these aligned throughout the resources.

This also means it should be used in places where kubernetes components use these labels as means for matching resources (Deployment/Pod for example)

Also, please check all other places where labels are used (exclude CRDs for now)

File: helm/keda/templates/12-keda-deployment.yaml [hunk 2]
```diff
@@ -5,23 +5,18 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     app: {{ .Values.operator.name }}
```
Comment (ubergesundheit) [inline helm/keda/templates/12-keda-deployment.yaml:7]:
```suggestion
```

File: helm/keda/templates/22-metrics-deployment.yaml [hunk 3]
```diff
@@ -5,16 +5,14 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     app: {{ .Values.operator.name }}-metrics-apiserver
```
Comment (ubergesundheit) [inline helm/keda/templates/22-metrics-deployment.yaml:7]:
```suggestion
```

**Pull request merged**